cli/package.json

```diff
@@ -43,13 +43,14 @@
     "user-home": "^2.0.0"
   },
   "devDependencies": {
+    "ava": "^2.0.0",
     "babel-eslint": "~10.0.1",
     "eslint": "~5.16.0",
     "eslint-config-prettier": "~4.3.0",
     "eslint-plugin-prettier": "~3.1.0",
-    "mocha": "~6.1.4",
-    "must": "~0.13.4",
-    "prettier": "~1.17.1"
+    "prettier": "~1.17.1",
+    "proxyquire": "^2.1.0",
```

Comment:
I evaluated a bunch of mocking/stubbing tools when approaching #118 and went with `testdouble`. It's more beginners friendly, can do both module replacement and different ways of stubbing (meaning you can replace both proxyquire and sinon with just testdouble) and comes with nice assertions and very human understandable assertion errors

Reply:
@gribnoysup honestly, I disagree. I love my tools, have been working with them for years (with sinon at least). They are mature (not something that was written for the presentation). I personally don't need to evaluate anything, I know what was working for years will work here as well. Sinon is not hard, it's pretty much as beginner friendly (I remember picking it up in a matter of hours when I first started learning it way back). Stub, spy and mock indeed have their own functions and that is a good thing. Regarding `testdouble` solving everything I personally don't believe it's an advantage. I like when one lib solves one particular problem. 

We need other members of this now growing community to decide what would be comfortable to use for testing for everyone. I'd like to stick with sinon + proxyquire.

I guess this PR is now paused :green_apple:

Reply:
I don't think this is the way to approach solving this problem. Of course we all love our own tools. Familiarity feels nice. Choosing new tools and libraries isn't about our own preferences, but rather a collective preference. 

There's some really good work going on in #118 and since @chrisdickinson has already done _some_ review, I would assume that using `testdouble` is fine.

Reply:
@gribnoysup @olingern  `testdouble` is now in use :tada: